src/bin/tectonic.rs

```diff
@@ -177,95 +206,16 @@ fn inner(
-             .takes_value(true))
-        .arg(Arg::with_name("web_bundle")
-             .long("web-bundle")
-             .short("w")
-             .value_name("URL")
-             .help("Use this URL find resource files instead of the default")
-             .takes_value(true))
-        .arg(Arg::with_name("only_cached")
-             .short("C")
-             .long("only-cached")
-             .help("Use only resource files cached locally"))
-        .arg(Arg::with_name("outfmt")
-             .long("outfmt")
-             .value_name("FORMAT")
-             .help("The kind of output to generate")
-             .possible_values(&["pdf", "html", "xdv", "aux", "format"])
-             .default_value("pdf"))
-        .arg(Arg::with_name("makefile_rules")
-             .long("makefile-rules")
-             .value_name("PATH")
-             .help("Write Makefile-format rules expressing the dependencies of this run to <PATH>"))
-        .arg(Arg::with_name("pass")
-             .long("pass")
-             .value_name("PASS")
-             .help("Which engines to run")
-             .possible_values(&["default", "tex", "bibtex_first"])
-             .default_value("default"))
-        .arg(Arg::with_name("reruns")
-             .long("reruns")
-             .short("r")
-             .value_name("COUNT")
-             .help("Rerun the TeX engine exactly this many times after the first"))
-        .arg(Arg::with_name("keep_intermediates")
-             .short("k")
-             .long("keep-intermediates")
-             .help("Keep the intermediate files generated during processing"))
-        .arg(Arg::with_name("keep_logs")
-             .long("keep-logs")
-             .help("Keep the log files generated during processing"))
-        .arg(Arg::with_name("synctex")
-             .long("synctex")
-             .help("Generate SyncTeX data"))
-        .arg(Arg::with_name("hide")
-             .long("hide")
-             .value_name("PATH")
-             .multiple(true)
-             .number_of_values(1)
-             .help("Tell the engine that no file at <PATH> exists, if it tries to read it"))
-        .arg(Arg::with_name("print_stdout")
-             .long("print")
-             .short("p")
-             .help("Print the engine's chatter during processing"))
-        .arg(Arg::with_name("chatter_level")
-             .long("chatter")
-             .short("c")
-             .value_name("LEVEL")
-             .help("How much chatter to print when running")
-             .possible_values(&["default", "minimal"])
-             .default_value("default"))
-        .arg(Arg::with_name("outdir")
-             .long("outdir")
-             .short("o")
-             .value_name("OUTDIR")
-             .help("The directory in which to place output files [default: the directory containing INPUT]"))
-        .arg(Arg::with_name("INPUT")
-             .help("The file to process, or \"-\" to process the standard input stream")
```

Comment:
It wasn't immediately obvious to me where the help strings had gone,
but now I see that they go into the doc comment.

Reply:
Agreed; this is one of those implicit things about `structopt`.